onyo/commands/cat.py

```diff
@@ -9,23 +9,26 @@
 logger = logging.getLogger('onyo')
 
 
-def build_cat_cmd(file):
-    this_asset = file
+def build_cat_cmd(files):
+    list_of_cat_commands = []
+    problem_str = ""
     onyo_repository_dir = os.environ.get('ONYO_REPOSITORY_DIR')
-    if not os.path.isfile(this_asset) and onyo_repository_dir is not None:
-        this_asset = os.path.join(onyo_repository_dir, this_asset)
-    if not os.path.isfile(this_asset):
-        logger.warning(file + " does not exist.")
-        return None
-    return "cat \"" + this_asset + "\""
+    for file in files:
+        if os.path.isfile(file):
+            list_of_cat_commands.append("cat \"" + file + "\"")
```

Comment:
Not a blocker for this PR, but I'm unaware of anything fancy that `cat` itself might be doing.

In general, we should stand on the shoulders-of-giants-written-in-C. Let git, etc al do the heavy lifting, and trust them. But it's not clear to me what `cat` might be giving us, and if it would be better to simply open and dump the contents of files in Python, and remove the overhead of calling a subprocess.